README.md

```diff
@@ -0,0 +1,79 @@
+# SystemLink Client API Documentation
+
+This repository contains getting started guides, examples, and reference
+documentation for the SystemLink Client NuGet packages. These packages support
+.NET Standard 2.0, meaning they support both .NET Core 2.0 and .NET Framework
```

Comment:
Could we make the split the second sentence into two sentences? 
"These packages support .NET Standard 2.0. Therefore, they support both .NET Core 2.0 and .NET Framework 4.6.1 with Visual Studio 2017 or later."